src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post">
```

Comment:
following code style recommendations
```suggestion
      <form
        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
        method="post"
      >
```